metadata.yaml

```diff
@@ -17,4 +17,13 @@ subordinate: true
 requires:
   juju-info:
     interface: juju-info
-    scope: container
\ No newline at end of file
+    scope: container
+resources:
+  amazon-efs-utils-deb:
+    type: file
+    filename: amazon-efs-utils.deb
+    description: |
+      The Debian package to install Amazon helper utils to mount an EFS volume.
+      This can be fetched and provided to the charm from https://github.com/aws/efs-utils.
```

Comment:
The https://github.com/aws/efs-utils doesn't include any deb package from the release page from what I saw. (Please correct me if I was mistaken). If that's the case, people have to build by themselves by following the instruction.
https://docs.aws.amazon.com/efs/latest/ug/installing-amazon-efs-utils.html#installing-other-distro

Reply:
If that is the case we need to clearly document in the README what is necessary for that or point to the AWS documentation.

Reply:
The description is for the resource which needs to be a debian package and this is the reason why I had named the attribute explicitly with the suffix `-deb`.  Also the github link and the aws doc link are exactly the same content wise apart from the latter being hosted on aws. The instructions for building the package are clearly mentioned in their readme. I can change the link if required.

Reply:
Lets link to the official AWS documentation instead of Github and say something like

`Please refer to the official AWS documentation [here](...) on how to obtain the package.`